### PR TITLE
docs(console): add the troubleshooting section in expo guide

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Add the troubleshooting section in the expo integration guide

image

## Checklist

- [ ] ~`.changeset`~
- [ ] ~unit tests~
- [ ] ~integration tests~
- [ ] ~necessary TSDoc comments~

### CODE REVIEW

File: packages/console/src/assets/docs/guides/native-expo/README.mdx [hunk 1]
```diff
@@ -169,4 +173,28 @@ const Content = () => {
...
+  // ...
+  resolver: {
+    unstable_enablePackageExports: true,
+  },
+};
+
+module.exports = config;
+```
+
+This error indicates that `@logto/rn` package not being able to resolve the `@logto/client/shim` module.
```
Comment (charIeszhao) [inline packages/console/src/assets/docs/guides/native-expo/README.mdx:192]: nit: `not being able to resolve` reads casual and it's more like oral English to me here. Better change it to `cannot resolve` or `is not able to resolve`.

**Review by charIeszhao — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]
LGTM, also need to update docs?

**Comment by simeng-li** [comment]
> LGTM, also need to update docs?

Yes, there is a docs open PR

File: packages/console/src/assets/docs/guides/native-expo/README.mdx (hunk 1 above)
Comment (simeng-li) [reply packages/console/src/assets/docs/guides/native-expo/README.mdx]: updated

**Pull request merged**